packages/services/f-braze-adapter/src/services/utils/filterByBrandName.js

```diff
@@ -1,2 +1,7 @@
-const filterByBrands = (cards, brands) => cards.filter(({ brand }) => !brand || brands.includes(brand));
+const filterByBrands = (cards, brands) => cards.filter(({ brand }) => {
+    if (brands.length > 0) {
+        return !brand || brands.includes(brand);
```

Comment:
Ah, the condition of this form makes more sense from a brand perspective - if a card isn't relevant to a brand then it should pass the filter, which is slightly different business logic than that for: `enabledCardTypes` 👍🏽